k6/tests/findDuplicates100kRegistrations.js

```diff
@@ -0,0 +1,62 @@
+import { check, fail, sleep } from 'k6';
+import { Counter } from 'k6/metrics';
+
+import { registrationPV } from '../helpers/registration-default.data.js';
+import loginModel from '../models/login.js';
+import RegistrationsModel from '../models/registrations.js';
+import resetModel from '../models/reset.js';
+
+const registrationsModel = new RegistrationsModel();
+const resetPage = new resetModel();
+const loginPage = new loginModel();
+
+const duplicateNumber = 17; // '17' leads to 131k registrations
+const resetScript = 'nlrc-multiple';
+const programId = 2;
+
+// At the time of implementation, the request duration was 12s on the server and 3s on the local machine for 130k registrations and about 8k duplicates
```

Comment:
Was this aligned with PO as acceptance criterium?

Reply:
yes